imap_processing/cdf/config/imap_swe_l1a_variable_attrs.yaml

```diff
@@ -0,0 +1,206 @@
+# <=== Coordinates ===>
+spin_angle:
+  CATDESC: 180 measurements per quarter cycle. See VAR_NOTES for details
+  FIELDNAM: Spin Angle
+  FILLVAL: -9223370000000000000
+  FORMAT: I3
+  LABLAXIS: Angle
+  SCALE_TYP: linear
+  UNITS: " "
+  VALIDMAX: 179
+  VALIDMIN: 0
+  VAR_NOTES: Combination of energy and spin angle. 180 measurements per quarter cycle.
+    At this point, we don't know about energy or spin angle information.
+  VAR_TYPE: support_data
+polar_angle:
+  CATDESC: This is angle of 7 CEMs
+  FIELDNAM: Polar Angle
+  FILLVAL: -9223370000000000000
+  FORMAT: I1
+  LABLAXIS: Angle
+  SCALE_TYP: linear
+  UNITS: int
+  VALIDMAX: 6
+  VALIDMIN: 0
+  VAR_NOTES: " "
+  VAR_TYPE: support_data
+# TODO: how to avoid repeating this?
+epoch:
+  CATDESC: Time, number of nanoseconds since J2000 with leap seconds included
+  FIELDNAM: epoch
+  LABLAXIS: epoch
+  FILLVAL: -9223372036854775808
+  FORMAT: " " # Supposedly not required, fails in xarray_to_cdf
+  VALIDMIN: -9223372036854775808
+  VALIDMAX: 9223372036854775807
+  UNITS: ns
+  VAR_TYPE: support_data
+  SCALETYP: linear
+  MONOTON: INCREASE
+  TIME_BASE: J2000
+  TIME_SCALE: Terrestrial Time
+  REFERENCE_POSITION: Rotating Earth Geoid
+
+# <=== Data Variables ===>
```

Comment:
It is nit-picky/personal preference, but I would find it to be a bit more readable if there was a blank line between each data variable